pulsar-broker/src/main/java/com/yahoo/pulsar/broker/BrokerData.java

```diff
@@ -0,0 +1,52 @@
+package com.yahoo.pulsar.broker;
```

Comment:
I am not sure why this diff is not outdated: the copy-right is present if you look at Files Changed.